shared_model/interfaces/query_responses/account_asset_response.hpp

```diff
@@ -36,25 +33,9 @@ namespace shared_model {
        */
       virtual const AccountAsset &accountAsset() const = 0;
 
-      /**
-       * Stringify the data.
-       * @return string representation of data.
-       */
-      std::string toString() const override {
-        return detail::PrettyStringBuilder()
-            .init("AccountAssetResponse")
-            .append(accountAsset().toString())
-            .finalize();
-      }
+      std::string toString() const override;
```

Comment:
Why did you remove docs? 📜

Reply:
I consulted with a couple of people and concluded that those comments were much too obvious and thus unnecessary. Do you think they should take place?

Reply:
Yeah! This documentation isn't looked useful. Just remove it.

Documentation of overloaded methods should contain useful details of implementation and/or changes in semantic.